public/stylesheets/rtl/main.css

```diff
@@ -1,140 +1,100 @@
-/* /////////////////////////////////////////////////////
```

Comment:
These are probably here because `gulp build` was not run before commit (there's no documentation of this and I didn't explain it, no blame). It's not strictly necessary at any rate.